<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>192</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetMaximumSize</enum>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="3" column="2">
     <widget class="QPushButton" name="pushButton3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="pushButtonMinus">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pushButton6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButton8">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton7">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButton9">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="pushButtonDivision">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="pushButtonMultiply">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButton1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pushButton2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="pushButton0">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="pushButtonDot">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="pushButtonEqual">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="pushButtonAdd">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="pushButtonCls">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>cls</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QLineEdit" name="lineEditResult">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Open 24 Display St</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Enter a equation to start!</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>309</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuStyle">
    <property name="title">
     <string>Style</string>
    </property>
    <addaction name="actionFusion"/>
    <addaction name="actionWindows"/>
    <addaction name="actionWindowsVista"/>
    <addaction name="separator"/>
    <addaction name="actionQdarkstyle"/>
    <addaction name="actionQdarkgraystyle"/>
    <addaction name="actionMyStyle"/>
    <addaction name="actionResetStyle"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuStyle"/>
  </widget>
  <action name="actionFusion">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fusion</string>
   </property>
  </action>
  <action name="actionWindows">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Windows</string>
   </property>
  </action>
  <action name="actionWindowsVista">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>WindowsVista</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionQdarkgraystyle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Qdarkgraystyle</string>
   </property>
  </action>
  <action name="actionQdarkstyle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Qdarkstyle</string>
   </property>
  </action>
  <action name="actionMyStyle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>MyStyle</string>
   </property>
  </action>
  <action name="actionResetStyle">
   <property name="text">
    <string>ResetStyle</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton0</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonDot</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonEqual</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonAdd</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>37</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton1</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>29</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>8</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonMinus</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton4</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>30</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton5</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton6</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonMultiply</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>282</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton7</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>23</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>8</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton8</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton9</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonDivision</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCls</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFusion</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_action_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionWindows</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_action_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionWindowsVista</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_action_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_action_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQdarkgraystyle</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_action_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQdarkstyle</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_action_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionMyStyle</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_action_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionResetStyle</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_action_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_button_clicked()</slot>
  <slot>on_action_triggered()</slot>
 </slots>
</ui>
